chimney/src/main/scala/io/scalaland/chimney/internal/compiletime/Results.scala

```diff
@@ -1,12 +1,35 @@
 package io.scalaland.chimney.internal.compiletime
 
+import io.scalaland.chimney.internal.{TransformerDerivationError as DerivationError, *}
+
 import scala.annotation.nowarn
 import scala.collection.compat.*
 import scala.util.control.NonFatal
 
 @nowarn("msg=The outer reference in this type test cannot be checked at run time.")
 private[compiletime] trait Results { this: Definitions =>
 
+  // TODO: consider expanding
+  trait Trace {
```

Comment:
Looks like something transformer-specific. Is it possible to decouple it from Result?